fenix/app/src/main/java/org/mozilla/fenix/settings/quicksettings/protections/ProtectionsView.kt

```diff
@@ -63,11 +67,21 @@ class ProtectionsView(
         binding.trackingProtectionDetails.setOnClickListener {
             interactor.onTrackingProtectionDetailsClicked()
         }
+        updateDividerVisibility()
+    }
+
+    private fun updateDividerVisibility() {
+        trackingProtectionDivider.isVisible = !(
+            !binding.trackingProtectionSwitch.isVisible &&
+                !binding.trackingProtectionDetails.isVisible &&
+                !binding.cookieBannerItem.isVisible
+            )
```

Comment:
```suggestion
        trackingProtectionDivider.isVisible = 
            binding.trackingProtectionSwitch.isVisible ||
                binding.trackingProtectionDetails.isVisible ||
                binding.cookieBannerItem.isVisible
```

This will be easy to understand.

Reply:
Done